control-station/src/components/SensorBoxes/Console.tsx

```diff
@@ -1,14 +1,32 @@
+import { useContext, useEffect, useRef } from "react";
 import "./SensorData.css";
+import PodContext from "@/services/PodContext";
 
 function Console() {
+	const { podData } = useContext(PodContext);
+	const listEndRef = useRef<HTMLLIElement | null>(null);
+
+	useEffect(() => {
+		if (listEndRef.current) {
+			listEndRef.current.scrollIntoView({ behavior: "smooth" });
+		}
+		console.log(podData.messages);
+	}, [podData.messages]);
+
 	return (
 		<div className="console">
 			<h2>Console</h2>
 			<ul className="console-list">
-				<li className="console-list-item">Start Sent</li>
-				<li className="console-list-item">Stop Sent</li>
-				<li className="console-list-item">Load Sent</li>
-				<li className="console-list-item">Force Stop Sent</li>
+				{podData.messages.map((prop, index) => (
+					<li
+						key={index}
+						className="console-list-item"
+						ref={index === podData.messages.length - 1 ? listEndRef : null}
+					>
+						{prop.timestamp} &nbsp;
+						{prop.message.toUpperCase()} State
```

Comment:
Question: is there a particular reason you added a non-breaking space here?
Suggestion [non-blocking]: displaying as a table with two columns may be more semantically appropriate

Reply:
Just to make sure there is always enough space and are readable